EnglishFilter/sections/antiadblock.txt

```diff
@@ -455,7 +455,7 @@ safemaru.blogspot.com###unblocker
 ! https://github.com/AdguardTeam/AdguardFilters/issues/32766
 videogreen.gq#@#.afs_ads
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38347
-zoobeeg.com#%#AG_abortOnPropertyRead('adBlockDetected');
+zoobeeg.com#$#.pub_300x250.pub_300x250m.pub_728x90.text-ad.textAd.text_ad.text_ads.text-ads.text-ad-links { display: block !important; }
```

Comment:
We have global CSS rule, please add domain to it.

Reply:
all these rules can be replaced by `zoobeeg.com##.swal2-container`. Easier - better.

Reply:
I remember we had problems in the past when we hid swal2-container (some Turkish sites). That modal is sometimes used for login/register etc.

Reply:
this site has players only.